docs/proposals/20210310-opt-in-autoscaling-from-zero.md

```diff
@@ -246,6 +297,8 @@ metadata:
     capacity.cluster-autoscaler.kubernetes.io/taints: "key1=value1:NoSchedule,key2=value2:NoExecute"
 ```
 
+If the `capacity.cluster-autoscaler.kubernetes.io/labels` annotation specifies a label that would otherwise be generated from the `capacity` or `node-info` annotations, the autoscaler will use the label defined in `capacity.cluster-autoscaler.kubernetes.io/labels`, overriding any labels produced by processing the other annotations.
```

Comment:
TBH i struggle to follow, might be because this sentence enters into autoscaler implementation details.

What about simplifying by writing what takes precedence between `capacity` or `node-info` on the template vs annotations on MachineSet or MachineDeployment?

Reply:
Changed to 

>The `capacity.cluster-autoscaler.kubernetes.io/labels` annotation takes precedence over other sources when conflicts occur.
For instance, if the `kubernetes.io/arch` label is specified in `capacity.cluster-autoscaler.kubernetes.io/labels`, its value supersedes that specified by `node-info.cluster-autoscaler.kubernetes.io/architecture`.